src/main/java/org/matsim/episim/EpisimConfigGroup.java

```diff
@@ -159,6 +161,8 @@
 	private int daysInfectious = 4;
 	private DistrictLevelRestrictions districtLevelRestrictions = DistrictLevelRestrictions.no;
 	private String districtLevelRestrictionsAttribute = "";
+	private ContagiousOptimization contagiousContainerOptimization = ContagiousOptimization.disable;
+	private ReportTimeUse reportTimeUse = ReportTimeUse.enable;
```

Comment:
I think this is something that can be disabled by default

Reply:
In most other enums we are using `yes` and `no`. Just to be consistent, we may also change it here.